docs/running-clusterfuzzlite/gitlab.md

```diff
@@ -0,0 +1,229 @@
+---
```

Comment:
Can you add a todo to this page to add a link to this doc [here](https://github.com/google/clusterfuzzlite/blob/85bb71ba22c90698f38e01171b6b7efd46ac47ec/docs/clusterfuzzlite.md?plain=1#L41) [here],(https://github.com/google/clusterfuzzlite/blob/1583ba6edd86e16bd57bf85f3dea93e813335972/docs/running_clusterfuzzlite.md?plain=1#L181) and a mention [here](https://github.com/google/clusterfuzzlite/blob/85bb71ba22c90698f38e01171b6b7efd46ac47ec/README.md?plain=1#L34), once this page is finalized?

Reply:
> Can you add a todo to this page ?

What do you mean ?
I added the links